diagnostic.md

```diff
@@ -35,7 +35,8 @@ Record your responses inside the fenced code blocks below each question.
     ```
 
     ```md
-    <!-- your response here -->
+    The first group of routes are nested and their folder structure will be nested as well.
+    In the second route, since it is not nested you have to explicitly state the full path.
```

Comment:
You are right that nested route definitions, do require a nested file structure, and that these two route definitions are identical  URL paths.
What else do you know is different?
One of the biggest distinctions is where Ember will render the content.  The nested `'products'/'product'` instructs Ember to render `app/products/product/template.hbs` in the `outlet` of the parent `app/products/template.hbs`.  The non-nested `'product'` instructs Ember to render `app/product/template.hbs` in the context of the application template.